backend/src/main/scala/cromwell/backend/wdl/ReadLikeFunctions.scala

```diff
@@ -9,7 +9,24 @@ import wdl4s.values._
 
 import scala.util.{Failure, Success, Try}
 
-trait ReadLikeFunctions extends PathFactory { this: WdlStandardLibraryFunctions =>
+trait FileSizeLimitationConfig {
+  val readLinesLimit = 128000
+  val readBoolLimit = 5
```

Comment:
Maybe make this one a little higher to take care of any trailing newline characters, etc?